CONTRIBUTING.md

```diff
@@ -0,0 +1,132 @@
+
+> **Note:** Before filing an issue, please check both open and closed issues to see if the problem has already been reported.
+
+#### Documentation
+
+We are happy to welcome contributions from anyone willing to improve documentation by adding missing information or making it more consistent and coherent.
+
+
+## Dev Environment
+
+* Install [Node.js](https://nodejs.org/en/) 
+
+* [Fork the repo and clone your fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo)
+
+* Get all dependencies
+  ```
+  npm install
+  ```
+* Add the upstream source for being able to sync main project changes back into your fork. For example, to configure an upstream remote repository for the `eslint-plugin-jsx-a11y` fork, run:
+  ```
+  git remote add upstream git@github.com:jsx-eslint/eslint-plugin-jsx-a11y.git 
+  ```  
+* Run the tests and the build
+  ```
+  npm test
+  ``` 
+* Make and submit changes to the project source files following our [pull request submission workflow](#pull-requests)
+	
+
+## Pull requests
+
+Create a new branch
+
+```
+git checkout -b issue1234
+```
+
+Commit the changes to your branch, including a coherent commit message that follows our [standards](#commit-messages)
+
+```
+git commit -a
+```
+
+Before sending the pull request, make sure your code is running on the latest available code by rebasing onto the upstream source
+
+```
+git fetch upstream
+git rebase upstream/main
+```
+
+Verify your changes
+
+```
+npm run test
+```
+
+Push your changes
+
+```
+git push origin issue1234
+```
+
+Send the [pull request](https://docs.github.com/en/pull-requests), make requested changes, and get merged 
+
+## Commit messages
+
+* Limit the first line of the commit message (message summary) to 72 characters or less
+* Start the commit message with an applicable tag:
+    * `Breaking` for anything semver-major
+    * `New` for anything semver-minor
+    * `Deps` for anything that updates runtime or peer deps
+    * `Dev deps` for anything that affects dev deps only   
+    * `Readme` for changes to the README only
+    * `Docs` for changes to API documentation
+    * `Robustness` for a semver-patch refactor that makes the package more robust against environment breakage (f.ex., modifying built-in JS methods)
+    * `Performance` for a code change that improves performance 
+    * `Refactor` for no observable changes (semver-patch)
+    * `Actions` for changes to Github Actions workflows only
+    * `Tests` for anything to do with testing that's not just updating dev deps
+    * `Eslint` for config and/or code changes that affect linting
+    * `Meta` for changes to project infra not covered by the above (changes to package.json, github config files, etc.)
+    * `Patch` for anything semver-patch not covered by the above   
+* Use the present tense and imperative mood when providing a description of what you did
+* If your PR addresses an issue, reference it in the body of the commit message
```

Comment:
it might be better to link to the gist; then we don't have to repeat this content and i only have one place to keep up to date.